### PR TITLE
Update composer deps for 2.x dev kit

### PR DESCRIPTION
**Pull request by covex-nn**
I am targeting this branch, because this is BC.

Closes #524

## Changelog

```markdown
### Changed
- Require highest versions of phpcr/phpcr and phpcr/phpcr-utils
```

## Subject

A job with `COMPOSER_FLAGS="--prefer-lowest"` is failing because it require too much time for `composer update`

Also, `symfony-cmf/testing` v2.1 package is not compatible with current `2.x` (i guess)

### CODE REVIEW

**Comment by covex-nn** [comment]
Travis build was not started because base branch was `2.x-dev-kit` first =(

**Comment by covex-nn** [comment]
@kunicmarko20 travis is green :tada:

File: composer.json [hunk 1]
```diff
@@ -44,10 +44,12 @@
         "dantleech/glob-finder": "^1.0",
         "jackalope/jackalope-jackrabbit": "^1.3",
         "matthiasnoback/symfony-dependency-injection-test": "^1.1",
+        "phpcr/phpcr": "^2.1",
+        "phpcr/phpcr-utils": "^1.4.0",
         "sonata-project/core-bundle": "^3.8",
         "symfony-cmf/resource": "^1.0",
         "symfony-cmf/resource-bundle": "^1.0",
-        "symfony-cmf/testing": "^2.0.1",
+        "symfony-cmf/testing": "2.0.*",
```
Comment (kunicmarko20) [inline composer.json]: shouldn't you keep `^`?

Comment (covex-nn) [reply composer.json]: A new `2.1.0` version of `symfony-cmf/testing` was released. And travis is failing with `^2.0.1` constraint, see https://travis-ci.org/sonata-project/SonataDoctrinePhpcrAdminBundle/builds/395047521 - i manually triggered that build. After #521 it should be updated back to `^2.1`

I want to propose changes to `dev-kit` for #512, but i want to fix travis first

Comment (ElectricMaxxx) [reply composer.json]: when not moving, we should keep "old" testing version here

**Comment by greg0ire** [comment]
Thanks @covex-nn !

**Pull request merged**